src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -384,6 +384,19 @@ public static String formatDateForInstructorCoursesPage(Instant instant, ZoneId
         return formatInstant(instant, zoneId, "d MMM yyyy");
     }
 
+    /**
+     * Formats a date stamp from an {@code instant} for the instructor's feedback sessions page.
+     * Example: 5 May 2017
+     * TODO: rename formatDateForInstructorCoursesPage and remove this method to avoid duplicated code
+     *
+     * @param instant the instant to be formatted
+     * @param zoneId  the time zone to calculate local date
+     * @return the formatted date stamp string
+     */
+    public static String formatDateForInstructorSessionsPage(Instant instant, ZoneId zoneId) {
+        return formatInstant(instant, zoneId, "d MMM yyyy");
```

Comment:
Can you open an issue to combine this method and the previous one into something like `formatDateForInstructorPages`? You do not need to implement it in this PR.

Reply:
Sure

Reply:
Should we merge this first since the method isn't in `master` yet? After that I'll create an issue.